src/parser/jobs/drk/modules/OGCDDowntime.js

```diff
@@ -0,0 +1,14 @@
+import ACTIONS from 'data/ACTIONS'
+import CooldownDowntime from 'parser/core/modules/CooldownDowntime'
+
+export default class OGCDDowntime extends CooldownDowntime {
+	// eslint-disable-next-line no-magic-numbers
```

Comment:
'aite that bothers me that lint is triggering on this.
google would suggest eslint doesn't support class props like this which is fucking annoying.